packages/@react-spectrum/list/test/ListView.test.js

```diff
@@ -288,42 +338,144 @@ describe('ListView', function () {
     });
 
     describe('ArrowUp', function () {
-      it('should not change focus from first item', function () {
+      it('should not wrap focus', function () {
         let tree = renderListWithFocusables();
-        let start = getCell(tree, 'Foo');
+        let start = getRow(tree, 'Foo');
         act(() => start.focus());
         moveFocus('ArrowUp');
         expect(document.activeElement).toBe(start);
       });
 
       it('should move focus to above row', function () {
         let tree = renderListWithFocusables();
-        let start = getCell(tree, 'Bar');
-        let end = getCell(tree, 'Foo');
+        let start = getRow(tree, 'Bar');
+        let end = getRow(tree, 'Foo');
+        act(() => start.focus());
```

Comment:
like, this one should be `userEvent.tab()` i think

Reply:
The start.focus was a easy shortcut to get focus onto a row that isn't the starting row. I can see about using a press instead